stubs/assertpy/assertpy/extracting.pyi

```diff
@@ -0,0 +1,10 @@
+from collections.abc import Iterable as _Iterable
+from typing import Any
+from typing_extensions import Self, TypeAlias
+
+Iterable: TypeAlias = _Iterable[Any]
+str_types: tuple[type[str]]
```

Comment:
I would omit this too, it's just Python 2/3 compatibility stuff.

Reply:
Sure, I removed `str_types`, `Iterable`, and one `xrange`!